istio/metadata.csv

```diff
@@ -1,327 +1,356 @@
+istio.sidecar_injection.skip_total,count,,request,,[OpenMetrics V1 and V2 and Istio v1.5+] Total number of skipped sidecar injection requests,0,istio,
+istio.mesh.request.duration.milliseconds.sum,count,,millisecond,,[OpenMetrics V1 and V2 and Istio v1.5+] Total sum of observed values for duration of requests in ms. This metric is sent as gauge by default in OpenMetrics V1.,0,istio,
+istio.mesh.request.duration.milliseconds.count,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total count of observed values for duration of requests. This metric is sent as gauge by default in OpenMetrics V1.,0,istio,
+istio.mesh.tcp.connections_closed.total,gauge,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total closed connections,0,istio,
+istio.mesh.tcp.connections_opened.total,gauge,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total opened connections,0,istio,
+istio.mesh.tcp.received_bytes.total,gauge,,byte,,[OpenMetrics V1 and V2 and Istio v1.5+] Size of total bytes received during request in case of a TCP connection,0,istio,
+istio.mesh.tcp.send_bytes.total,gauge,,byte,,[OpenMetrics V1 and V2 and Istio v1.5+] Size of total bytes sent during response in case of a TCP connection,0,istio,
+istio.mesh.request.count.total,count,,request,,[OpenMetrics V1 and V2 and Istio v1.5+] The number of requests as monotonic count,0,istio,
+istio.mesh.request.duration.milliseconds.count.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total count of observed values for duration of requests as monotonic count,0,istio,
+istio.mesh.request.duration.milliseconds.sum.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total sum of observed values for duration of requests as monotonic count,0,istio,
+istio.mesh.request.size.count.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Count of observed request sizes as monotonic count,0,istio,
+istio.mesh.request.size.sum.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Sum of observed request sizes as monotonic count,0,istio,
+istio.mesh.response.size.count.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Count of observed response size as monotonic count,0,istio,
+istio.mesh.response.size.sum.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Sum of observed response size as monotonic count,0,istio,
+istio.mesh.tcp.connections_closed.total.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total closed connections as monotonic count,0,istio,
+istio.mesh.tcp.connections_opened.total.total,count,,,,[OpenMetrics V1 and V2 and Istio v1.5+] Total opened connections as monotonic count,0,istio,
+istio.mesh.tcp.received_bytes.total.total,count,,byte,,[OpenMetrics V1 and V2 and Istio v1.5+] Size of total bytes received during request in case of a TCP connection as monotonic count,0,istio,
+istio.mesh.tcp.send_bytes.total.total,count,,byte,,[OpenMetrics V1 and V2 and Istio v1.5+] Size of total bytes sent during response in case of a TCP connection as monotonic count,0,istio,
```

Comment:
I'm doubtful these are emitted with V2. Do you have an example output from an environment?

i noticed you listed some metrics as available from V2 but they're actually not. I'm not sure if you actually saw these metrics from your test environment? These were added as part of a config option specific _only_ to V1 https://github.com/DataDog/integrations-core/pull/5707